examples/js/app.ts

```diff
@@ -36,4 +40,38 @@ autocomplete({
     recentSearchesPlugin,
     querySuggestionsPlugin,
   ],
+  getSources() {
+    return [
```

Comment:
We can use this product template (converted to DOM) for the demo:

https://github.com/algolia/autocomplete.js/blob/29ca383835c2efea0a4efbdddba329682c467409/examples/js/app.tsx#L56-L96

Reply:
Should I still keep the `empty` template? I feel like this is great to have it in an example

Reply:
Yes!